src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -571,8 +580,11 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
   override def backendType = BackendType.JES
 
   def gcsAuthFilePath(descriptor: WorkflowDescriptor): String = {
-    val wfPath = workflowGcsPath(descriptor)
-    s"$wfPath/auth.json"
+    // If we are going to upload an auth file we need a valid GCS path passed via workflow options.
+    val bucket = descriptor.workflowOptions.get(AuthFilePathOptionKey) getOrElse {
+      throw new Throwable(s"$AuthFilePathOptionKey has not been found in workflow options.")
+    }
```

Comment:
It actually still fails relatively early, because the first time we call this is when trying to write the auth file which is during `initializeForWorkflow`.